tests/Unit/DataStructures/Test_Normalize.cpp

```diff
@@ -0,0 +1,117 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+
+#include "DataStructures/Tensor/Expressions/Normalize.hpp"
+
+TEST_CASE("Unit.DataStructures.Tensor.EuclideanNormalize",
+          "[DataStructures][Unit]") {
+  const size_t npts = 8;
+  DataVector one(npts, 1.0);
+  DataVector two(npts, 2.0);
+  DataVector three(npts, 3.0);
+  DataVector four(npts, 4.0);
+  DataVector five(npts, 5.0);
+  DataVector twelve(npts, 12.0);
+  DataVector thirteen(npts, 13.0);
```

Comment:
I see little reason to use a length-8 `DataVector` with constant entries here (and in the other tests as well). I think a length-1 `DataVector` is sufficient to verify the normalization calculation, and will be faster.

A cleaner approach, used in the Determinant test, is to have most of the testing done with `Tensor<double>`, and one or two tests checking `Tensor<DataVector>`.